_posts/2024-03-25-new-sponsoring-tiers.md

```diff
@@ -0,0 +1,43 @@
+---
+title: New sponsoring tiers
+summary: We refurbish the tiers to provide more to our sponsors
+author: beta-ziliani,straight-shoota
+---
+
+We made several changes to the sponsor packages offered on the [OpenCollective sponsoring page](https://www.opencollective.com/crystal-lang).
+
+The sponsoring tiers and their perks hadn't been updated in a while and we felt it was time to freshen up.
+Particularly, we want to offer more differentiated opportunities for expressing thanks to our sponsors.
+
+The general structure of tiers hasn't changed. It's a bit more refined now with clearer naming.
+We consider three groups of sponsorship levels with different target audiences:
+
+1. **Backer:** Individuals and small organizations who love Crystal and want to support it.
```

Comment:
The images are a bit bulky and disrupt the text. Maybe trim the cards?
<img width="240" alt="image" src="https://github.com/crystal-lang/crystal-website/assets/5183634/b00fb478-42aa-43ef-afab-ed9e4e6d2060">

Or even more to stick to the essentials?
<img width="235" alt="image" src="https://github.com/crystal-lang/crystal-website/assets/5183634/ed9c0cc7-fe05-444a-9098-e074d3cbaf81">